api/prisma/migrations/20231116075406_add_proxy_location/migration.sql

```diff
@@ -0,0 +1,22 @@
+/*
+  Warnings:
+
+  - Added the required column `location` to the `proxies` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- AlterTable
+ALTER TABLE "proxies" ADD COLUMN     "location" TEXT NOT NULL;
+
+-- CreateTable
```

Comment:
@nick-bisonai Let's remove the generation of `L2AggregatorPair` and also addition of the foreign key on it. We can keep everything until line 8.

Reply:
updated based on the feedback :)